en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/L/Light.md

```diff
@@ -12,3 +30,5 @@ Until the spell ends, an object no larger than 10 feet in any dimension emits br
 Light emanating from the target may be any color.
 Completely covering or enclosing the target with something that is not transparent blocks the light.
 The spell ends when you use an action to dismiss it or if you cast it again.
+
+If you target an object held or worn by a hostile creature, that creature must succeed on a Dexterity saving throw to avoid the spell.
```

Comment:
This was included in the WotC version but was missing in the A5e version. I think it's worth having.